src/main/java/com/useful/ucars/ucars.java

```diff
@@ -170,6 +171,11 @@ public void run() {
 	@Override
 	public void onEnable() {
 		plugin = this;
+		
+		//bStats
+		int pluginId = 11901; // <-- Replace with the id of your plugin!
+        Metrics metrics = new Metrics(this, pluginId);
```

Comment:
Please can we keep bStats out of the master uCars repo, I'm not a huge fan of non-consensual analytics data being sent to a third party - especially through a plugin ID managed by somebody else